src/services/DynamoDBService.ts

```diff
@@ -113,11 +113,11 @@ export class DynamoDBService {
   /**
    * Retrieves the ongoing activity for a given staffId
    * @param staffId - staff id for which to retrieve activity
-   * @returns Promise<PromiseResult<DocumentClient.QueryOutput, AWSError>>
+   * @returns Promise<IActivity[]>
```

Comment:
Can you rename IActivity to Activity instead?

Reply:
Updated as per suggestion